src/transforms/dedupe.rs

```diff
@@ -18,8 +18,6 @@ pub enum FieldMatchConfig {
     MatchFields(Vec<String>),
     #[serde(rename = "ignore")]
     IgnoreFields(Vec<String>),
-    #[serde(skip)]
-    Default,
```

Comment:
Hit error `FieldMatchConfig::Default` cannot be serialized. I fixed this and also remove runtime panic.